go.mod

```diff
@@ -9,20 +9,21 @@ require (
 	github.com/fatih/structs v1.1.0
 	github.com/go-clix/cli v0.1.1
 	github.com/gobwas/glob v0.2.3
-	github.com/google/go-cmp v0.5.2-0.20200818193711-d2fcc899bdc2
+	github.com/google/go-cmp v0.5.4
 	github.com/google/go-jsonnet v0.17.0
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/imdario/mergo v0.3.9 // indirect
 	github.com/karrick/godirwalk v1.15.5
 	github.com/pkg/errors v0.9.1
 	github.com/posener/complete v1.2.3
+	github.com/prometheus/prometheus v1.8.2-0.20210215121130-6f488061dfb4
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/objx v0.2.0
-	github.com/stretchr/testify v1.5.1
+	github.com/stretchr/testify v1.7.0
 	github.com/thoas/go-funk v0.4.0
-	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	gopkg.in/yaml.v2 v2.2.8
-	gopkg.in/yaml.v3 v3.0.0-20191010095647-fc94e3f71652
-	k8s.io/apimachinery v0.19.2
+	golang.org/x/crypto v0.0.0-20201208171446-5f87f3452ae9
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
`gopkg.in/yaml.v2` was pinned to `v2.2.8` in https://github.com/grafana/tanka/pull/386 on purpose, becuase upgrading brings whitespace changes. @captncraig is this still relevant, or can we perform the upgrade by now?